api/base/filters.py

```diff
@@ -368,7 +368,7 @@ def get_filtered_queryset(self, field_name, params, default_queryset):
         if isinstance(field, ser.SerializerMethodField):
             return_val = [
                 item for item in default_queryset
-                if self.FILTERS[params['op']](self.get_serializer_method(field_name)(item), params['value'])
+                if self.FILTERS[params['op']](self.get_serializer_method(field_name)(item), int(params['value']))
```

Comment:
'Cause really, who's going to send their size via API in TB or something dumb like that? Not even [pick your least favorite provider] does that.